app/views/steps/supporting_evidence/edit.html.erb

```diff
@@ -0,0 +1,65 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<%= tag.div class: "govuk-grid-row" do -%>
```

Comment:
where there is no logic I am inclinded to use raw html... so the above line becomes:
```
<div class="govuk-grid-row">
```